charts/argo-cd/templates/argocd-applicationset/clusterrolebinding.yaml

```diff
@@ -13,12 +13,4 @@ subjects:
   - kind: ServiceAccount
     name: {{ template "argo-cd.applicationSet.serviceAccountName" . }}
     namespace: {{ include "argo-cd.namespace" . }}
-{{- $namespaces := index .Values.configs.params "applicationsetcontroller.namespaces" -}}
-{{- range $namespace := (split "," $namespaces) }}
-  {{- if $namespace }}
-  - kind: ServiceAccount
-    name: {{ include "argo-cd.applicationSet.serviceAccountName" $ }}
-    namespace: {{ $namespace | trim | quote }}
-  {{- end }}
-{{- end }}
```

Comment:
We can use regex at `applicationsetcontroller.namespaces` and it causes issue when users use regex  like `argocd-appsets-*`. So I am wondering how we can provide the additional permission binding... 🤔 
Is it reasonable to rely on users to bind SA and ClusterRole (and create SA in each namespace) by themselves ?